src/Components/Facility/ConsultationForm.tsx

```diff
@@ -559,8 +559,10 @@ export const ConsultationForm = (props: any) => {
     return <Loading />;
   }
 
+  const twoColDiv = "flex gap-4 flex-nowrap flex-shrink-0";
```

Comment:
The need for this feels slightly unnecessary. I don't think you need to put it into a variable.And the name doesn't seem to have anything to do with the classes